kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaConsumerTests.java

```diff
@@ -65,6 +68,39 @@ public void shouldDeleteRecords() {
         .hasSize(0);
   }
 
+  @Test
+  public void shouldIncreasePartitionsUpTo10() {
+    var topicName = UUID.randomUUID().toString();
+    webTestClient.post()
+        .uri("/api/clusters/{clusterName}/topics", LOCAL)
+        .bodyValue(new TopicCreation()
+            .name(topicName)
+            .partitions(1)
+            .replicationFactor(1)
+            .configs(Map.of())
+        )
+        .exchange()
+        .expectStatus()
+        .isOk();
+
+    PartitionsIncreaseResponse response = webTestClient.patch()
+        .uri("/api/clusters/{clusterName}/topics/{topicName}/partitions",
+            LOCAL,
+            topicName)
+        .bodyValue(new PartitionsIncrease()
+            .totalPartitionsCount(10)
+        )
+        .exchange()
+        .expectStatus()
+        .isOk()
+        .expectBody(PartitionsIncreaseResponse.class)
+        .returnResult()
+        .getResponseBody();
+
+    assert response != null;
+    Assertions.assertEquals(10, response.getTotalPartitionsCount());
+  }
+
```

Comment:
Do you mind if we add one more GET request here to get the updated topic details? This is just to double-check that your changes are applied successfully